goagen/codegen/helpers_test.go

```diff
@@ -0,0 +1,37 @@
+package codegen_test
+
+import (
+	"testing"
+
+	"github.com/goadesign/goa/goagen/codegen"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestHelpers(t *testing.T) {
+	var (
+		stringTestA = "testAa"
```

Comment:
Why not inline those to the `Expect` call below? It would simplify reading the tests a bit.